### PR TITLE
get-started: move app repo

### LINKED ISSUE #17837: Dockerfile question
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Other

### Description

The documentation ** may ** be incorrect. I am learning and I am unsure. 
I ran the git clone for getting-started. I am using VS Code 2022. 
Clicking the Windows tab it says to: Create an empty file name Dockerfile. 
However, there is a Dockerfile, already. Is this information to be overwritten with the #2; add the following contents......
Do I add the following, or overwrite? 

# syntax=docker/dockerfile:1
   
FROM node:18-alpine
WORKDIR /app
COPY . .
RUN yarn install --production
CMD ["node", "src/index.js"]
EXPOSE 3000

thank you, 
Stephanie

### Location

https://docs.docker.com/get-started/02_our_app/

### Suggestion

_No response_

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Issue: The sample app is inside the getting-started tutorial repo. There have been several issues related to users running the commands or creating files in the wrong directory, and being confused by the Dockerfile in the repo root.

Fix: Create a new standalone repo for the sample app.

### Related issues (optional)
ENGDOCS-1510
Fixes #17837

### CODE REVIEW

File: get-started/02_our_app.md [hunk 1]
```diff
@@ -24,22 +24,28 @@ To complete this guide, you'll need the following:
...
    ```
 
-2. View the contents of the cloned repository. Inside the `getting-started/app` directory you should see `package.json` and two subdirectories (`src` and `spec`).
+2. View the contents of the cloned repository. You should see the following files and subdirectories.
 
-    ![Screenshot of Visual Studio Code opened with the app loaded](images/ide-screenshot.png){: style="width:650px;margin-top:20px;"}
-    {: .text-center }
+   ```
+   ├── getting-started-app/
+   │ ├── pacakge.json
```
Comment (stephaurelio) [inline get-started/02_our_app.md:37]: Small typo: 
```suggestion
   │ ├── package.json
```

**Review by stephaurelio — COMMENTED** [review]
Nice work! This should make the guide more clear. I really like the inclusion of the file structure so they can verify their project looks the same. Just left two small comments, ones a typo if you want to commit that real quick then I'll circle back to approve!

File: get-started/02_our_app.md [hunk 2]
```diff
@@ -50,9 +56,9 @@ To build the [container image](../get-started/overview.md/#docker-objects){:targ
 
     In the terminal, run the following commands listed below.
 
-    Change directory to the `app` directory. Replace `/path/to/app` with the path to your `getting-started/app` directory.
+    Change directory to the `getting-started-app` directory. Replace `/path/to/getting-started-app` with the path to your `getting-started-app` directory.
```
[the remark this answers is not in the document]
Comment (craig-osterhout) [reply get-started/02_our_app.md:59]: I'll update the wording to account for users who may already be in the correct directory. Running `cd` again won't affect them. If you mean remove the `cd` codeblock, then I'm hesitant.

Some users skip over paragraphs straight to the codeblocks. In the previous step, we don't explicitly say to use the CLI to view the contents, so they may using the GUI and then their pwd will still be in the directory above where they ran the clone command. We could move the `cd` command to the previous step, but I don't think anything is gained.

File: get-started/02_our_app.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ Now that you have an image, you can run the application in a [container](../get-
 
    The `-d` flag (short for `--detach`) runs the container in the background.
    The `-p` flag (short for `--publish`) creates a port mapping between the host and the container.
-   The `-p` flag takes a string value in the format of `HOST:CONTAINER`,
+   The `-p` flag take a string value in the format of `HOST:CONTAINER`,
```
Comment (stephaurelio) [inline get-started/02_our_app.md]: I think the previous version was correct:
```suggestion
   The `-p` flag takes a string value in the format of `HOST:CONTAINER`,
```

**Review by stephaurelio — COMMENTED** [review]
Nice ✨ I just noted one typo that snuck in.

**Review by stephaurelio — APPROVED** [review]
LGTM!

**Pull request merged**